src/components/organisms/SearchBar.vue

```diff
@@ -46,67 +55,46 @@ export default defineComponent({
     <v-card-title>
       Sample Project
     </v-card-title>
-    <TabToolBar
+    <!-- Will be needed in second iteration -->
+    <!-- <TabToolBar
       v-model="activeTab"
       :items="tabData"
       color="blue-grey darken-2"
       flat
-    />
+    /> -->
     <v-form @submit.prevent="$emit('input', inputs)">
-      <v-row
+      <!-- Will be needed in second iteration -->
+      <!-- <LatLongForm
         v-if="activeTab === 0"
-        no-gutters
-        justify="center"
-      >
-        <v-col
-          cols="5"
-          class="mr-3 mt-3"
-        >
-          <v-text-field
-            v-model.number="inputs.longitude"
-            label="Longitude(deg)"
-            outlined
-          />
-        </v-col>
-        <v-col
-          cols="5"
-          class="ml-3 mt-3"
-        >
-          <v-text-field
-            v-model.number="inputs.latitude"
-            label="Latitude(deg)"
-            outlined
-          />
-        </v-col>
-        <v-col
-          cols="9"
-          class="pl-1 pr-4"
-        >
-          <v-text-field
-            v-model.number="inputs.height"
-            label="Height (m)"
-            outlined
-          />
-        </v-col>
-        <v-btn
-          type="submit"
-          color="teal accent-4"
-          min-height="56px"
-          max-width="56px"
-          min-width="unset"
-        >
-          <v-icon
-            large
-          >
-            mdi-crosshairs-gps
-          </v-icon>
-        </v-btn>
-      </v-row>
+      />
+      <BoundingBoxForm
+        v-if="activeTab === 1"
+      /> -->
+      <ToolBar
+        text="Geo Json"
+        color="blue-grey darken-2"
+        flat
+      />
+      <GeoJsonForm />
```

Comment:
Would it make sense for the contents of `ToolBar` to be part of `GeoJsonForm` instead of a seperate component? It seems like it's more of a header then a toolbar, unless of course things will be added to the toolbar in the future.